rclpy/rclpy/executors.py

```diff
@@ -755,6 +755,7 @@ def __init__(self, num_threads: int = None, *, context: Context = None) -> None:
             warnings.warn(
                 'MultiThreadedExecutor is used with a single thread.\n'
                 'Use the SingleThreadedExecutor instead.')
+        self.futures = []
```

Comment:
this meant to be internal use only?

```suggestion
        self._futures = []
```